### PR TITLE
Fix error message, arguments were misplaced, restructure unit tests

### PR DESCRIPTION
**Pull request by panslava**
What a miss :D

/assign @code-elinka @aojea

### CODE REVIEW

File: pkg/loadbalancers/l4_test.go [hunk 1]
```diff
@@ -1651,6 +1654,63 @@ func TestEnsureInternalDualStackLoadBalancer(t *testing.T) {
...
+	}
+
+	for _, tc := range testCases {
+		tc := tc
+
+		t.Run(tc.desc, func(t *testing.T) {
+			t.Parallel()
+
+			ipFamilies := []v1.IPFamily{v1.IPv4Protocol, v1.IPv6Protocol}
+			svc := test.NewL4ILBDualStackService(8080, v1.ProtocolTCP, ipFamilies, v1.ServiceExternalTrafficPolicyTypeCluster)
```
Comment (code-elinka) [inline pkg/loadbalancers/l4_test.go:1684]: Could you please remind why it is only ExternalTrafficPolicy=Cluster, not Local? Probably it jumed out of my head.

File: pkg/loadbalancers/l4netlb_test.go [hunk 2]
```diff
@@ -614,22 +617,6 @@ func TestEnsureIPv6ExternalLoadBalancerCustomSubnet(t *testing.T) {
 	svc.Annotations = result.Annotations
 	assertDualStackNetLBResourcesWithCustomIPv6Subnet(t, l4NetLB, nodeNames, "another-subnet")
 
-	// change to internal IPv6 subnet and expect error
```
Comment (code-elinka) [inline pkg/loadbalancers/l4netlb_test.go:617]: Could you please add a comment about the logic of https://github.com/kubernetes/ingress-gce/pull/2216/files#diff-43bde103b548631af39e388b8ebe203e970ed5f4957ea6be8d710746230fb3d1L569 ? It seems to be long

File: pkg/loadbalancers/l4netlb_test.go [hunk 3]
```diff
@@ -647,6 +634,63 @@ func TestEnsureIPv6ExternalLoadBalancerCustomSubnet(t *testing.T) {
 	assertDualStackNetLBResourcesDeleted(t, l4NetLB)
 }
 
+func TestDualStackNetLBBadCustomSubnet(t *testing.T) {
```
Comment (code-elinka) [inline pkg/loadbalancers/l4netlb_test.go:637]: Do we have a test with wrong subnet name only for dualstack?

**Comment by code-elinka** [comment]
Logic of the test looks good. I would also move new tests closer to original subnet tests so it's easier to catch that some of the cases are out of standard test.

File: pkg/loadbalancers/l4_test.go (hunk 1 above)
Comment (panslava) [reply pkg/loadbalancers/l4_test.go:1684]: it just does not matter for this test case, we can not really test every possible combination, and etp=Local does not change logic in this case, 

we could omit that, but NewL4ILBDualStackService requires to specify

File: pkg/loadbalancers/l4netlb_test.go (hunk 2 above)
Comment (panslava) [reply pkg/loadbalancers/l4netlb_test.go:617]: added a comment

**Comment by panslava** [comment]
applied comments, moved TestDualStackILBBadSubnet close to TestEnsureInternalLoadBalancerCustomSubnet

ready for re-review

File: pkg/loadbalancers/l4netlb_test.go (hunk 3 above)
Comment (code-elinka) [reply pkg/loadbalancers/l4netlb_test.go:642]: Okay, I got it now, yeah, my question was if we want to test that behaviour for ipv4 only as well. Thank you for explaining it

File: pkg/loadbalancers/l4_test.go (hunk 1 above)
Comment (code-elinka) [reply pkg/loadbalancers/l4_test.go]: got it!

**Pull request merged**